static/app/components/iconCircledNumber.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from '@emotion/styled';
+
+type IconCircledNumberProps = {
+  number: number;
+  size?: number;
```

Comment:
Can we use the standard `theme` values for size (`xs | sm | md | ...`)?
https://github.com/getsentry/sentry/blob/349f28bcf53fa031c05afe6d18c5f08f0c0d7c12/static/app/utils/theme.tsx#L686-L693